Extension fails to bootstrap vcpkg on Windows

I am running VSCode on Windows with the project [Open-CMSIS-Pack/vscode-get-started](https://github.com/Open-CMSIS-Pack/vscode-get-started) containing a `vcpkg-configuration.json` in its root.

vcpkg fails with the following output:
```
> Using vcpkg executable at ".vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.0-win32-x64\assets\platform\win32-x64\vcpkg.exe".
> Using VCPKG_ROOT at "AppData\Roaming\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root".
> Activating environment at "vscode-get-started\vcpkg-configuration.json".
> Version of vcpkg used to create VCPKG_ROOT: "2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75".
> Running vcpkg with arguments ["version"].
vcpkg package management program version 2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75

See LICENSE.txt for license information.
> Bundled version of vcpkg: "2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75".
> Running vcpkg with arguments ["activate","--project","vscode-get-started","--json","AppData\\Local\\Temp\\d5da59d60e5fb3221061526b6de69ca4"].
warning: vcpkg-ce ('configure environment') is experimental and may change at any time.
Downloading vcpkg-artifacts bundle 2023-01-24...
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2023-01-24/vcpkg-ce.tgz
write_contents("AppData\Roaming\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\node_modules\ce-sha.txt"): no such file or directory
> Failed with exit code "1".
> Failed to activate environment at "vscode-get-started\vcpkg-configuration.json".
```

Any ideas what's going wrong? Running the same on Linux (WSL Ubuntu) works as expected.

Let me share an update with you after updating the VSCode extension to 0.6.230315001

```powershell
$vcpkg = "$Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg.exe"

$Env:VCPKG_ROOT = "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root"

rm -Recurse $Env:VCPKG_ROOT

& $vcpkg --version
vcpkg package management program version 2023-03-14-105f7f7e8a5ea10dfac9874ff9d1a6bacba5f454

& $vcpkg bootstrap-standalone --debug
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from $Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg-bundle.json
[DEBUG] Failed to open: $Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
Downloading standalone bundle 2023-03-14.
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2023-03-14/vcpkg-standalone-bundle.tar.gz
[DEBUG] Trying to hash $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\vcpkg-standalone-bundle.tar.gz.39648.part
[DEBUG] $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\vcpkg-standalone-bundle.tar.gz.39648.part has hash 32024d3d703aaa5961e8072ab883537a858885c501223530285926f919bad67003d40ba35686eb9d24673c4b030bb289a82d515c7bdf9931e9249d873b2b2abd
[DEBUG] Found path: C:\Windows\system32\tar.exe
[DEBUG] Found path: c:\tools\git\usr\bin\tar.exe
[DEBUG] CreateProcessW("C:\Windows\system32\tar.exe" xzf "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\vcpkg-standalone-bundle.tar.gz")
[DEBUG] cmd_execute() returned 0 after 173839 us
[DEBUG] D:\a\_work\1\s\src\vcpkg\commands.bootstrap-standalone.cpp(44):
[DEBUG] Time in subprocesses: 173839us
[DEBUG] Time in parsing JSON: 0us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 4498us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 1.1683323 s (1153884us)

& $vcpkg activate --debug
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from $Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg-bundle.json
[DEBUG] Failed to open: $Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Using scripts-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\scripts
[DEBUG] Using builtin-ports: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\ports
[DEBUG] Using installed-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\installed
[DEBUG] Using buildtrees-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\buildtrees
[DEBUG] Using packages-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\packages
[DEBUG] Using vcpkg-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root
[DEBUG] Using scripts-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\scripts
[DEBUG] Using builtin-registry: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\versions
[DEBUG] Using downloads-root: $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads
warning: vcpkg-artifacts is experimental and may change at any time.
A suitable version of node was not found (required v16.15.1) Downloading portable node 16.15.1...
Downloading node...
https://nodejs.org/dist/v16.15.1/node-v16.15.1-win-x64.7z->$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\node-v16.15.1-win-x64.7z
Downloading https://nodejs.org/dist/v16.15.1/node-v16.15.1-win-x64.7z
[DEBUG] Trying to hash $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\node-v16.15.1-win-x64.7z.19268.part
[DEBUG] $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\node-v16.15.1-win-x64.7z.19268.part has hash 7ec4bfe2ea6034e1461e306b6372d62c0c5d1060c453ba76a73a5cec38ac26b5952a744caa9071455329caa58eb0a96d26c68854c8915c17610ff27b0cf2c1cf
Extracting node...
[DEBUG] Found path: C:\tools\cmake\bin\cmake.exe
[DEBUG] CreateProcessW("C:\tools\cmake\bin\cmake.exe" --version)
[DEBUG] ReadFile() finished with GetLastError(): 109
[DEBUG] 1000: cmd_execute_and_stream_data() returned 0 after   111073 us
[DEBUG] CreateProcessW("C:\tools\cmake\bin\cmake.exe" -E tar xzf "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\node-v16.15.1-win-x64.7z")
[DEBUG] cmd_execute() returned 0 after 3192535 us
[DEBUG] CreateProcessW("$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\tools\node-16.15.1-windows\node-v16.15.1-win-x64\node.exe" --version)
[DEBUG] ReadFile() finished with GetLastError(): 109
[DEBUG] 1001: cmd_execute_and_stream_data() returned 0 after  1521218 us
[DEBUG] vcpkg-artifacts base path: $Env:USERPROFILE\AppData\Local\vcpkg\artifacts-2023-03-14
Downloading vcpkg-artifacts bundle 2023-03-14...
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2023-03-14/vcpkg-ce.tgz
[DEBUG] Trying to hash $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\vcpkg-ce-2023-03-14.tgz.19268.part
[DEBUG] $Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\vcpkg-ce-2023-03-14.tgz.19268.part has hash fb2c2f568b563a88d2ebf2b29ced491e8109134529c477629e5e1dde89f52bf8a60d9b5ce8285f260cc94b7c100704128d3724bbda523edb57233dbda5a2c02a
[DEBUG] CreateProcessW("$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\tools\node-16.15.1-windows\node-v16.15.1-win-x64\node.exe" "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\tools\node-16.15.1-windows\node-v16.15.1-win-x64\node_modules\npm\bin\npm-cli.js" --force install --no-save --no-lockfile --scripts-prepend-node-path=true --silent "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\vcpkg-ce-2023-03-14.tgz")
[DEBUG] cmd_execute() returned 0 after 10720866 us
[DEBUG] CreateProcessW("$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\tools\node-16.15.1-windows\node-v16.15.1-win-x64\node.exe" "$Env:USERPROFILE\AppData\Local\vcpkg\artifacts-2023-03-14\node_modules\vcpkg-ce" activate --debug --debug --z-telemetry-file "$Env:USERPROFILE\AppData\Local\Temp\vcpkg\2e1b9aa0-c6dd-420a-be4f-5768f1f42e07_artifacts_telemetry.txt" --vcpkg-root "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root" --z-vcpkg-command "$Env:USERPROFILE\.vscode\extensions\ms-vscode.vscode-embedded-tools-0.6.230315001-win32-x64\assets\platform\win32-x64\vcpkg.exe" --z-vcpkg-artifacts-root "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads\artifacts" --z-vcpkg-downloads "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\downloads" --z-vcpkg-registries-cache "$Env:USERPROFILE\AppData\Local\vcpkg\registries" --z-next-previous-environment "$Env:USERPROFILE\AppData\Local\Temp\vcpkg\a7ea3a81-e809-49e2-8aff-133115fc84ce_previous_environment.txt" --z-global-config "$Env:APPDATA\Code\User\globalStorage\ms-vscode.vscode-embedded-tools\vcpkg\root\vcpkg-configuration.json")
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '$Env:USERPROFILE\AppData\Local\vcpkg\artifacts-2023-03-14\node_modules\vcpkg-ce'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
[DEBUG] cmd_execute() returned 1 after 338353 us
[DEBUG] Failed to open: $Env:USERPROFILE\AppData\Local\Temp\vcpkg\2e1b9aa0-c6dd-420a-be4f-5768f1f42e07_artifacts_telemetry.txt
[DEBUG] Telemetry file couldn't be read: no such file or directory
[DEBUG] D:\a\_work\1\s\src\vcpkg\commands.activate.cpp(11):
[DEBUG] Time in subprocesses: 15884045us
[DEBUG] Time in parsing JSON: 0us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 17196us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 18.2089487 s (18192679us)
```

Now, it is causing an exception in vcpkg.

--- a/scripts/bootstrap.ps1
+++ b/scripts/bootstrap.ps1
@@ -45,7 +45,7 @@ while (!($vcpkgRootDir -eq "") -and !(Test-Path "$vcpkgRootDir\.vcpkg-root"))
 
 Write-Verbose "Examining $vcpkgRootDir for .vcpkg-root - Found"
 
-$versionDate = '2023-03-29'
+$versionDate = '2023-04-07'
 if ($env:PROCESSOR_ARCHITECTURE -eq 'ARM64' -or $env:PROCESSOR_IDENTIFIER -match "ARMv[8,9] \(64-bit\)") {
     & "$scriptsDir/tls12-download-arm64.exe" github.com "/microsoft/vcpkg-tool/releases/download/$versionDate/vcpkg-arm64.exe" "$vcpkgRootDir\vcpkg.exe"
 } else {

--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -126,23 +126,23 @@ fi
 
 # Choose the vcpkg binary to download
 vcpkgDownloadTool="ON"
-vcpkgToolReleaseTag="2023-03-29"
+vcpkgToolReleaseTag="2023-04-07"
 if [ "$UNAME" = "Darwin" ]; then
     echo "Downloading vcpkg-macos..."
-    vcpkgToolReleaseSha="f83ec768200c6ece26555aba8d25c17063fed27ef7744f42eaa7772fa55a0f6187a2a4bae6fa4a72a23f6bb8590e056b13d4215e340d4e3d42de030b9e36a721"
+    vcpkgToolReleaseSha="2a2805aa251a9523311c813e9e68896f18805fed4ace301c603f4afac3940b27d4ef1d3b4572c8f93ce4d13ac613720275b9e21e39cc5d19115d2fc849bfe2bb"
     vcpkgToolName="vcpkg-macos"
 elif [ "$vcpkgUseMuslC" = "ON" ]; then
     echo "Downloading vcpkg-muslc..."
-    vcpkgToolReleaseSha="f81ccaffe8ba2e775277312296581c5cb37f5f659f09cf3c99c07346ff1e73e9e4ac42aeed31e0db408b49ab0114bc947db5e26c4f24a57b13a6ad0c12d0b423"
+    vcpkgToolReleaseSha="d8dc48e6dc866f4ebe3919c0bf1377769d6c6f1ad2dab7fc09da9b26f7dac3ab3b06affb585ece7ba72e2fdeacdc77b8df31ad08cff49e4c060b7647fc1cc22a"
     vcpkgToolName="vcpkg-muslc"
 elif [ "$ARCH" = "x86_64" ]; then
     echo "Downloading vcpkg-glibc..."
-    vcpkgToolReleaseSha="4fea1d3f36a1895b5535da9595871f80900c58aac288ffb7939290f13d268ee82ba8e358675e2f191ab6564536a809de42b3819fec300b0195a8d8e8513d0051"
+    vcpkgToolReleaseSha="f26aaf5f503b9fd0a8b206230df19af966390d7087a9f3342f24c2d5e73f1f1bd81cba2695a89c87015ec822bd41bf836c2a9ef9f0e11acc61d6d9593aa8fae9"
     vcpkgToolName="vcpkg-glibc"
 else
     echo "Unable to determine a binary release of vcpkg; attempting to build from source."
     vcpkgDownloadTool="OFF"
-    vcpkgToolReleaseSha="3c80824f31775aac981d13dd5e4aab8565f4ec9b45440cd8d9b2b76d4131071fbcc7170979475f436a355c7f079d41d6b140ec84465c7ff4fac9ca26d20d0a30"
+    vcpkgToolReleaseSha="89dc32154e22f6a51fa18a8ea22c16263533546469f73cdb4ee168ab7c68ebc2b2a93177284dc03ecf2d2ca438f9fd583def48844788942e98ea45027e8ec2fa"
 fi
 
 # Do the download or build.

